jenkins/verify-chef.sh

```diff
@@ -10,24 +10,39 @@ export PATH
 EMBEDDED_BIN_DIR=/opt/$PROJECT_NAME/embedded/bin
 export EMBEDDED_BIN_DIR
 
+# If we are on Mac 10.11 our symlinks are located under /usr/local/bin
+# otherwise they are under /usr/bin
+if [ -a /usr/bin/sw_vers ]; then
+  platform_version=`sw_vers | awk '/^ProductVersion:/ { print $2 }' | cut -d. -f1,2`
+
+  if [ $platform_version == "10.11" ]; then
```

Comment:
The use of `/usr/local/bin` should be true for all versions of Mac OS X now:
https://github.com/chef/omnibus-chef/blob/7d655f62dbb71c556002a9c1c4693e0edc1aed8a/package-scripts/chef/postinst#L36-L38